tests/unitests/test_schema.py

```diff
@@ -23,6 +23,7 @@ def sample_package() -> Packages:
 def sample_feedstock_output(sample_feedstock, sample_package) -> FeedstockOutputs:
     return FeedstockOutputs(
         id="output_1",
+        path="/path/to/output_1",
```

Comment:
Hopefully we can uncouple this in the future. I do believe it shouldn't be part of the database tables. After all it's just an implementation detail in the original repo-database.